_dashboards/drag-drop-wizard.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Drag and drop wizard (experimental feature)
```

Comment:
suggest to keep visualization type names parallel in left nav: 
(e.g. Region map visualizations) 
maybe change title to "Wizard visualizations (experimental feature)"